tools/docker/Dockerfile.test_spack

```diff
@@ -42,9 +42,9 @@ RUN apt-get update -qq && apt-get install -qq --no-install-recommends \
 
 # Install a recent developer version of Spack.
 WORKDIR /opt/spack
-ARG SPACK_VERSION=501ee68606405ea2ac26d369a5139d75be88dac8
+ARG SPACK_VERSION=f75c1dd55815a787d6d69c2c1e79a7440c39edda
 RUN git init --quiet && \
-    git remote add origin https://github.com/spack/spack.git && \
+    git remote add origin https://github.com/RMeli/spack.git && \
```

Comment:
Switch back to upstream Spack when https://github.com/spack/spack/pull/48594 is merged.